test/io/ferguson/read-class-reorder.chpl

```diff
@@ -0,0 +1,29 @@
+/*class Base {
+}
+
+class Parent : Base {
+  var x: int;
+  var y: int;
+}
+*/
+
+class Child /*: Parent*/ {
```

Comment:
There's some leftover code here, do you want to copy it to a second test?  It seems like that would make a good edge case to me